etna/records/field_labels.py

```diff
@@ -0,0 +1,9 @@
+# format <record_field_name> : <human readable label name>
+human_readable_labels = {
```

Comment:
It's common in Python/Django for variables defined at the global level to be named in uppercase, e.g.: `HUMAN_READABLE_LABELS` 

The variable name is probably a bit more wordy than it needs to be (the comment above does a nice enough job of explaining what this is for - nice touch!!). Something like `FIELD_LABELS` would be fine.

Reply:
Thanks, this is done